examples/process-quarkus-example.yaml

```diff
@@ -1,17 +1,14 @@
 apiVersion: app.kiegroup.org/v1alpha1
-kind: KogitoApp
+kind: KogitoBuild
 metadata:
   name: example-quarkus
 spec:
-  build:
-    #envs:
-    # envs can be used to set variables during build
-    #- name: MY_CUSTOM_ENV
-    #  value: "my value"
-    gitSource:
-      contextDir: process-quarkus-example
-      uri: 'https://github.com/kiegroup/kogito-examples'
-    # set your maven nexus repository
-    #mavenMirrorURL: http://nexus3-nexus.apps-crc.testing/repository/maven-public/
```

Comment:
The `mavenMirrorURL` attribute is very important, could you please add it to the new example?

Reply:
added back